docs/getting-started/experiments/bayes_ab/index.md

```diff
@@ -0,0 +1,54 @@
+# Bayesian A/B Testing
+
+Bayesian A/B testing is compares two variants: treatment (e.g. a new feature) and control (e.g. an existing feature). The two variants are shown to users at random. Unlike frequentist A/B testing, this method requires you to set prior probabilities for the treatment and control arms, similarly to the bandit experiments. However, unlike the bandit methods, the posterior is computed at the _end_ of the experiment, and not with every observed outcome. Consequently, this means that the probability with which arms are served to the user are also fixed by the user-defined priors throughout the experiment.
+
+## What is Bayesian A/B testing?
+
+With A/B testing, you have 2 variants of a feature / implementation (one is ideally a baseline / existing feature that you want to compare the other against), with corresponding probability of achieving a desired outcome. You serve beneficiaries either variant at a random but _fixed_ probability throughout the experiment and observe the outcome of their interaction with it. You update these probabilities for each variant _after_ the experiment is over and all te observations are in.
```

Comment:
Do we restrict people from adding more than two variants? Can you add additional arms?

Reply:
This content is very similar to the one above. Can we maybe talk about _why_ or _when_ you'd use this above and get into the details here?

Reply:
Also, we can peek with bayesian A/B testing right? So you can still see which ARM is doing better. We just don't update sampling probabilities. I'd drop the last line since it's confusing - you have already said that sampling probabilities are fixed during the experiment